--- php-pear-PHP_Compat-1.3.1/PHP_Compat-1.3.1/Compat.php~	2004-11-23 10:25:40.000000000 +0100
+++ php-pear-PHP_Compat-1.3.1/PHP_Compat-1.3.1/Compat.php	2004-12-05 00:30:19.497058608 +0100
@@ -95,7 +95,7 @@
      */
     function loadVersion($version = null)
     {
-        require_once 'Compat/Components.php';
+        require_once 'PHP/Compat/Components.php';
         PHP_Compat::loadFunction('version_compare');
         $phpversion = phpversion();
 
